src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public interface Figure {
+    double getArea();
+
+    double getProperty();
+
+    void draw();
```

Comment:
The methods `getArea()` and `draw()` are conceptually different and should not be in the same interface. According to the checklist, all classes and interfaces should have a single purpose.